lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -917,6 +924,7 @@ export class NativeInteractionClient extends BaseInteractionClient {
                 ...request.tokenQueryParameters,
             },
             extendedExpiryToken: false, // Make this configurable?
+            keyId: request.popKid || "",
```

Comment:
Do we have to give this a default or can we simply not include if it doesn't exist?

Reply:
No we dont. I can remove that. Thanks!

Reply:
Removed!